### DISCUSSION TITLE
NetTopologySuite.IO.GeoJSON4STJ dependency version

### DISCUSSION THREAD

**Discussion by IldarKhayrutdinov** [Mapsui/Mapsui#1840] [General]
I noticed that `Mapsui.Avalonia` 4.0.0-beta.8 require `NetTopologySuite.IO.GeoJSON4STJ` (>= 2.1.1 && < 3.0.0)

https://www.nuget.org/packages/Mapsui.Avalonia/4.0.0-beta.8#dependencies-body-tab

Is it possible to upgrade to latest [3.0.0](https://www.nuget.org/packages/NetTopologySuite.IO.GeoJSON4STJ/3.0.0) version?

**Comment by pauldendulk** [comment]
It seems so. I created a PR https://github.com/Mapsui/Mapsui/pull/1843.
